### PR TITLE
add integration tests to PRs

### PR DESCRIPTION
**Pull request by dmou**
Includes ES service in integration tests for PRs.

### CODE REVIEW

**Comment by dmou** [comment]
hey @DGaffney are integration tests meant to be run from within the application container or the CI runner machine? is there a way we could set them to run them against `localhost:9200` rather than `elasticsearch:9200`?

**Comment by DGaffney** [comment]
Hello @dmou! I did not write the tests with the intent of running inside of either the application container or the CI runner machine (to be more clear, I simply wrote the tests with the intent of running them... somewhere. At some point. No further planning was made for this beyond that goal)- though it should be possible to get them to run anywhere. If you want to specify the ElasticSearch host as `localhost:9200` rather than `elasticsearch:9200`, you'll need to set the `es_host` environment variable. In the `ci-deploy-env.yml` file, you'd probably want to change the `docker run` command to something like this:

`docker run --rm -v $GITHUB_WORKSPACE/coverage:/app/coverage ${{ steps.build-image.outputs.image }} -e APP=fetch -e DEPLOY_ENV=travis -e GITHUB_TOKEN=${{ secrets.CONFIGURATOR_TOKEN }} -e es_host=localhost:9200 test_integration`

I may be going crazy but I feel like I may be remembering that setting a lowercased env var doesn't play well some times? If that's true just let me know and I can make changes so that you can specify it as an uppercased variable.

**Comment by dmou** [comment]
@DGaffney Got it, that's useful context! The `es_host` environment variable worked well, lowercase and all. Unfortunately for me, the issue seemed to have originated from Docker host network setup issues rather than test env settings. It was helpful for narrowing down the source of the issue though, so thanks!

File: .github/workflows/ci-deploy-env.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@ jobs:
   build-test-deploy:
     runs-on: ubuntu-latest
 
+    services:
+      elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch:6.8.9
```
Comment (huslage) [inline .github/workflows/ci-deploy-env.yml:16]: This needs to be 7.9.2

File: .github/workflows/ci-deploy-env.yml [hunk 2]
```diff
@@ -65,7 +71,7 @@ jobs:
         curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter && chmod +x ./cc-test-reporter
         ./cc-test-reporter before-build
         mkdir $GITHUB_WORKSPACE/coverage
-        docker run --rm -v $GITHUB_WORKSPACE/coverage:/app/coverage ${{ steps.build-image.outputs.image }} -e APP=fetch -e DEPLOY_ENV=travis -e GITHUB_TOKEN=${{ secrets.CONFIGURATOR_TOKEN }} test_unit
+        docker run --rm -v $GITHUB_WORKSPACE/coverage:/app/coverage ${{ steps.build-image.outputs.image }} -e APP=fetch -e DEPLOY_ENV=travis -e GITHUB_TOKEN=${{ secrets.CONFIGURATOR_TOKEN }} test_integration
```
Comment (huslage) [inline .github/workflows/ci-deploy-env.yml]: Do we need both tests or is integration enough?

**Review by huslage — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-deploy-env.yml (hunk 2 above)
Comment (infojunkie) [reply .github/workflows/ci-deploy-env.yml]: both please

**Comment by infojunkie** [comment]
Also, please add a Slack notification for test running results. This is what happens for Travis on other repos and is expected by devs.

**Review by huslage — APPROVED** [review]

**Comment by infojunkie** [comment]
Was the Slack integration added as requested?

**Pull request merged**